pkg/cgroups/cgroups.go

```diff
@@ -45,13 +45,19 @@ func Find() (Subsystems, error) {
 	scanner := bufio.NewScanner(mountInfo)
 	for scanner.Scan() {
 		lineSegs := strings.Fields(scanner.Text())
-		if lineSegs[len(lineSegs)-3] != "cgroup" {
+		if len(lineSegs) != 11 {
```

Comment:
Ah, looks like my first reading of the man page missed how the optional fields work. Yes, this is broken.  And I think it's not just one optional field but an arbitrary list of them.